DependencyInjection/FOSHttpCacheExtension.php

```diff
@@ -86,6 +86,9 @@ public function load(array $configs, ContainerBuilder $container)
 
         $container->setParameter($this->getAlias().'.compiler_pass.tag_annotations', $config['tags']['enabled']);
         if ($config['tags']['enabled']) {
+            // only load tag_invalidate if tags are enabled
+            $loader->load('tag_invalidate.xml');
```

Comment:
I’m in favour of moving all tag-related services into one XML file, as long as we need to either load everything or nothing in that file for the tagging enabled/disabled scenarios, which I think is the case.